libs/SalesforceSDKCommon/SalesforceSDKCommon.xcodeproj/project.pbxproj

```diff
@@ -32,6 +32,80 @@
 		82D53D541A3627EC00E46F8F /* NSData+SFSDKUtils.m in Sources */ = {isa = PBXBuildFile; fileRef = 82D53D521A3627EC00E46F8F /* NSData+SFSDKUtils.m */; };
```

Comment:
We need to mask the exported CocoaLumberjack symbols, either as part of the project configuration, or in an xcconfig file—I'd probably opt for the latter—so that developers can include their own CocoaLumberjack dependencies without ours interfering.  Basically, we need the following somewhere that will be picked up by the project configuration:

```obj-c
OTHER_CFLAGS = (
	"-DDDAbstractDatabaseLogger=SF_DDAbstractDatabaseLogger",
	"-DDDLog=SF_DDLog",
	"-DDDAbstractLogger=SF_DDAbstractLogger",
	"-DDDASLLogger=SF_DDASLLogger",
	"-DDDAbstractLogger=SF_DDAbstractLogger",
	"-DDDLogger=SF_DDLogger",
	"-DDDLogFileManager=SF_DDLogFileManager",
	"-DDDLogFileManagerDefault=SF_DDLogFileManagerDefault",
	"-DDDLogFileFormatterDefault=SF_DDLogFileFormatterDefault",
	"-DDDLogFormatter=SF_DDLogFormatter",
	"-DDDFileLogger=SF_DDFileLogger",
	"-DDDLogFileInfo=SF_DDLogFileInfo",
	"-DDDRegisteredDynamicLogging=SF_DDRegisteredDynamicLogging",
	"-DDDLogMessage=SF_DDLogMessage",
	"-DDDLoggerNode=SF_DDLoggerNode",
	"-DDDTTYLogger=SF_DDTTYLogger",
	"-DDDTTYLoggerColorProfile=SF_DDTTYLoggerColorProfile",
	"-DDDMultiFormatter=SF_DDMultiFormatter",
	"-DDDContextWhitelistFilterLogFormatter=SF_DDContextWhitelistFilterLogFormatter",
	"-DDDContextBlacklistFilterLogFormatter=SF_DDContextBlacklistFilterLogFormatter",
	"-DDDLoggingContextSet=SF_DDLoggingContextSet",
	"-DDDDispatchQueueLogFormatter=SF_DDDispatchQueueLogFormatter",
	"-DDDExtractFileNameWithoutExtension=SF_DDExtractFileNameWithoutExtension",
	"-DDDLogMessageCopyFile=SF_DDLogMessageCopyFile",
	"-DDDLogMessageCopyFunction=SF_DDLogMessageCopyFunction",
	"-DDDLogMessageOptions=SF_DDLogMessageOptions",
	"-DdoesAppRunInBackground=SF_doesAppRunInBackground",
);
```